### ISSUE TITLE
Test cases don't fully work

### ISSUE THREAD

**Issue by hazendaz** [mybatis/migrations-maven-plugin#17]
I noticed that all the test cases were not setup properly so I turned them on.  This resulted in 4 classes failing under the test package.  Possibly similar issues exists on integration tests but I didn't check those.

For regular tests, all get this error 'mybatis-migration_TEST.script (The system cannot find the file specified)'

Since some tests do pass, I'm leaving those on which will in turn show some code coverage which this library was clearly missing.

**Comment by h3adache** [comment]
I don't know what you mean about it not being setup properly. The tests used to run under junit 3 runs by using the name of the class and methods to figure out what tests exist. If this was changed, that is what broke tests.

**Comment by h3adache** [comment]
Where are these tests failing? I took out your ignores and tests are working fine?

**Comment by hazendaz** [comment]
Appears it would be me that messed them up.  I upgraded to junit 4 on ba09210feb78b481b446bd571e051799d2939b33 but didn't add @Test annotation which effectively turned all the tests off.

Do you know what creates 'mybatis-migration_TEST.script'? Or is that something that is supposed to be present?  If we can figure that out, I suspect all will run successfully.

**Comment by harawata** [comment]
> Do you know what creates 'mybatis-migration_TEST.script'?

It's [HSQLDB](http://hsqldb.org/doc/guide/ch01.html#N100F3).

**Comment by hazendaz** [comment]
Thanks.  I debugged it pretty deep and don't see where it's failing within the plugin.  It seems like it's up in migrations itself.  I'm using windows 10.  I also took out the ignores as @h3adache did and pushed to travis.  It built their successfully.  I also found one remaining legacy style maven usage so fixed that.  I'll push that up now.  I want to keep this open a bit longer tough in case anyone has any further insight.  

For reference, what machine types other than travis is this working on?
